fiftyone/core/runs.py

```diff
@@ -365,14 +365,18 @@ def save_run_info(cls, samples, run_info, overwrite=True):
         run_docs = getattr(dataset_doc, cls._runs_field())
         view_stages = [json_util.dumps(s) for s in samples.view()._serialize()]
 
-        run_docs[key] = RunDocument(
+        run_doc = RunDocument(
+            dataset_id=dataset_doc.id,
             key=key,
             version=run_info.version,
             timestamp=run_info.timestamp,
             config=deepcopy(run_info.config.serialize()),
             view_stages=view_stages,
             results=None,
         )
+        # run_doc.save()
```

Comment:
@brimoor should the run_doc be saved? This was originally in the saved_views pr but not in the develop head now